docs/01_setup.md

```diff
@@ -83,31 +90,35 @@ Copy the config example and update the fields as necessary:
 cp deploy-config-example.json deploy-config.json
 ```
 
-There are already some deployed contracts that could be reused, copy the `default-deployed-contracts.json` file if you need them to avoid deploying redundant contracts and save your gas fee.
+> [!IMPORTANT]
+> Make sure that you use the production zkeys (copy the correct generated coordinatorPubkey), set the `pollDuration` with the correct time in **seconds** and set `useQuadraticVoting` to false.
+
+### Deploy MACI Contracts
+
+There are already some deployed contracts that could be reused. Copy the `default-deployed-contracts.json` file if you need them to avoid deploying redundant contracts and save gas fees.
```

Comment:
I think the `default-deployed-contracts.json` only exists in the maci repo now? should we tell the reader here?

Reply:
Yes I think we should clarify this step is inside the MACI repo, I will add a step before: Go into the MACI repo

Reply:
Maybe for each heading we can specify to which repo they refer to, e.g:

### Deploy MACI Contracts (MACI Repo)